nixos/modules/services/matrix/mautrix-discord.nix

```diff
@@ -0,0 +1,469 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+# This module is an adaptation of the module for mautrix-meta in the same directory
+let
+  settingsFormat = pkgs.formats.yaml { };
+
+  upperConfig = config;
```

Comment:
Is this actually necessary? I don't remember the scoping rules well enough to remember if we can just reference `config` directly in the contexts that you are using `upperConfig`.